dtmi/azureiot/devkit/freertos/esp32azureiotkit-1.json

```diff
@@ -0,0 +1,203 @@
+{
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:azureiot:devkit:freertos:Esp32AzureIotKit;1",
```

Comment:
do we need `freertos` in the `@id`, I guess this same interface can be implemented with other platforms supported by ESP32

Reply:
That's a very good point. Do you suggest putting it one level up then?

Reply:
As you prefer,  just noticed we already have `dtmi/azureiot/devkit/freertos` 
/c @ryanwinter

Reply:
We could have generic models for the devkit, just not sure if that will cause problems in the future of we want a different esp32 model for a different sample.

Reply:
@rido-min , we will keep the model in this path for now since it's tied to the FreeRTOS middleware sample. Later we can move it if we decide to reuse the model.